packages/block-library/src/blocks/sidecar/scss/_mixins.scss

```diff
@@ -0,0 +1,123 @@
+// Default Settings for Sidecar Block
+:root {
+  --body-font-size: 16;
+  --offset-addon: 230;
+}
+
+// Default Layout Grid
+// fs - full start
+// ws - wide start
+// cs - content start
+// ce - content end
+// cc - content center
+// we - wide end
+// fe - full end
+// gs - gap start
+// ge - gap end
+// gcs - gap content start
+// gce - gap content end
+// The same thing applies for [cc] [ce].
+
+//noinspection ALL
+@mixin layout-grid {
+  display: var(--display-mode);
+  flex-direction: column;
+  
+  --default-sidebar-width: calc(var(--minimum-sidebar-width) * 0.5);
+  
+  --wds: var(--default-width-offset-sidebar, var(--default-sidebar-width));
+  --wde: var(--default-width-offset-sidebar, var(--default-sidebar-width));
+  
+  --full: calc((100vw - var(--theme-content-width-wide)) * 0.5);
+  
+  grid-template-columns: [fs] Max(0px, var(--editor-full, var(--full))) [ws] calc(var(--wds) * 2) [gs] var(--grid-gap) [cs] 1fr [gcs] calc(var(--grid-gap)/2) [cc] calc(var(--grid-gap)/2) [gce]  1fr [ce] var(--grid-gap) [ge] calc(var(--wde) * 2) [we] Max(0px, var(--editor-full, var(--full))) [fe];
+}
+
+// Default Layout Settings
+// --display-mode - Block default display mode. (eg. grid or flex)
+// --sticky-element-top-spacing - Used to position the sticky block.
+// In Rosa2 this value will be equal with site header height, in this way,
+// the sticky block will not be under the header.
+// --all-columns - Used for blocks that need to span on the entire grid.
+@mixin layout-settings {
+  --default-width-small-sidebar: 230px;
+  --default-width-medium-sidebar: 300px;
+  --default-width-large-sidebar: 500px;
```

Comment:
Wouldn't it be a better idea to have these variables defined in `ems` at all times?

Reply:
Usually yes. But we have chosen those minimum, to match some of the most used ad sizes. Considering the fact that ads are usually in px and fixed, I think this minimum should be also in px.